src/components/cax-companyData.tsx

```diff
@@ -123,12 +123,14 @@ export const CompanyDataCax = () => {
 
   useEffect(() => {
     const index = i18n.language === 'de' ? 0 : 1
-    countryList = countryList?.map((country) => ({
-      id: country.alpha2Code,
-      label:
-        country.countryName[index]?.value + ' (' + country.alpha2Code + ')',
-    }))
-    setCountryArr(countryList)
+    if (countryList && Array.isArray(countryList[0].countryName)) {
```

Comment:
App broke here if user came from step 2. So added this check.

Reply:
it will still break in the case if countryList is an empty array `[]` - so I suggest if we touch it cover that case as well

Reply:
@oyo 
Added a check for empty array too.